recipes/biopet-vcfstats/1.1/meta.yaml

```diff
@@ -0,0 +1,45 @@
+# Based on OpenJDK recipe in conda-forge
+# https://github.com/conda-forge/openjdk-feedstock/blob/master/recipe/meta.yaml
+# This file was automatically generated by the sbt-bioconda plugin.
+
+     
+test:
+  commands:
+  - biopet-vcfstats --version
+  - biopet-vcfstats --help
+package:
+  name: biopet-vcfstats
+  version: '1.1'
+source:
+  url: https://github.com/biopet/vcfstats/releases/download/v1.1/VcfStats-assembly-1.1.jar
+  sha256: f9ad47d5d840a99271d5f70b30046304330f1a1bf6e99756da5ecd2841da4a32
+build:
+  number: 0
+requirements:
+  run:
+  - openjdk
+  - python
+  build: []
+extra:
+  notes: 'biopet-vcfstats is a Java program that comes with a custom wrapper shell
+    script. By default ''no default java option'' is set in the wrapper. The command
+    that runs the program is ''biopet-vcfstats''. If you want to overwrite it you
+    can specify memory options directly after your binaries. If you have _JAVA_OPTIONS
+    set globally this will take precedence. For example run it with ''biopet-vcfstats
+    -Xms512m -Xmx1g''. '
+about:
+  home: https://github.com/biopet/vcfstats
+  license: MIT
+  summary: "Vcfstats is a tool that can generate metrics from a vcf file.\n\n - General\
```

Comment:
Can we improve the readability of the summary and remove line breaks. Summary is in most cases really short.

Reply:
Thank you for your swift reply.
Will do this!